### DISCUSSION TITLE
Trakt auth

### DISCUSSION THREAD

**Discussion by tigerkzr** [meisnate12/Plex-Meta-Manager#1236] [Q&A]
So since Trakt went down hard last week, I have been unable to reconnect.  It keeps giving me 503 errors. Anyone else encounter this and sort it out?

**Comment by mikenobbs** [comment]
When Trakt went down it took the API with it, and PMM needs the API. Trakt is back up but collections and connection in general won't work until Trakt restores their API next week 🙂

**Comment by tigerkzr** [comment]
I figured it was something like that. Didn't realize their api was lagging behind the site though. Thanks :)
